healthcheck/healthcheckreport.go

```diff
@@ -0,0 +1,32 @@
+package healthcheck
```

Comment:
Yup. I wasn't sure which case multiple word files should be in, changed it to lower camel case. Both files have been deleted